src/app/globals.css

```
@@ -3595,60 +3595,135 @@ html.custom-cursor-active * {
+
+.category-strip-arrow--left:active svg {
+  transform: scaleX(-1) translateX(3px);
+}
+
+.category-strip-arrow--right:active svg {
+  transform: translateX(3px);
+}
+
+.arrow-stroke {
+  fill: none;
+  stroke: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)